frontend/src/components/smart/CharacterList.vue

```diff
@@ -169,8 +169,8 @@ export default {
   },
 
   mounted() {
-    this.levelFilter = sessionStorage.getItem('character-levelfilter') || '';
-    this.elementFilter = sessionStorage.getItem('character-elementfilter') || '';
+    this.levelFilter = localStorage.getItem('character-levelfilter') || '';
+    this.elementFilter = localStorage.getItem('character-elementfilter') || '';
     if(this.isMarket) {
```

Comment:
revert code to this commit: 3fc33b4bcea38ed0bc37a9311d18c3fabac1ff49

Reply:
we want to be using the same kind of storage everywhere. if we're using session storage, we should continue to do so.

Reply:
I know, this is mistake I made change in this commit: 26907d3fe01922d7f9ebf8eb5056803a0af9be4e
So just recover to original code